app/tests/base_test.py

```diff
@@ -123,50 +150,25 @@ def check_stac_asset(self, expected, current, ignore=None):
         '''
         if ignore is None:
             ignore = []
-        self._check_stac_dictsubset('asset', expected, current, ignore=ignore + ['href'])
+        self._check_stac_dictsubset('asset', expected, current, ignore=ignore)
 
     def _check_stac_dictsubset(self, parent_path, expected, current, ignore=None):
         for key, value in expected.items():
             path = f'{parent_path}.{key}'
+
             # We need to remove the stac_extensions from here when BGDIINF_SB-1410 is implemented
```

Comment:
BGDIINF_SB-1410 has been implemented. So I guess stac_extensions could be removed here

Reply:
I though about this, but if we remove this line then we need to update the sample data and I was not sure about the status here. I'll create a ticket for this. This could be done as a ramp up for the other.